### ISSUE TITLE
NTR: Non-convulsive status epilepticus with coma

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5410]
"-Status epilepticus without prominent motor symptoms
---Non-convulsive status epilepticus without coma
---NTR: Non-convulsive status epilepticus with coma
label: Non-convulsive status epilepticus with coma
def: A type of status epilepticus without prominent motor symptoms and in the presence of coma.
comment: This includes the entity 'subtle status epilepticus' proposed by some authors as the late‚ burned-out stage of generalized convulsive status epilepticus characterized by (subtle rather than predominant) focal or multifocal myoclonic movements, coma, and pseudoperiodic epileptiform discharges against a slow low-voltage background on EEG.
source: PMID: 26336950
seeAlso: Subtle status epilepticus
http://webprotege.stanford.edu/RiCnLhdRh6VgSt0KNJ92ib: ILAE 2015
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney"
PMID:28276060: https://www.ncbi.nlm.nih.gov/pubmed/PMID:28276060

**Comment by pnrobinson** [comment]
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney

Would it be appropriate to merge the new terms in this subhierarchy with the existing non-convulsive terms? This would mean changing the label and definition of the old term to e.g this current one. This would only be appropriate if we are talking about the same thing though, otherwise, we might have to put the old terms in a "Pre-ILAE2018 legacy seizure" hierarchy?

https://hpo.jax.org/app/browse/term/HP:0031475

**Comment by pnrobinson** [comment]
Sorry, I see that you already assigned  https://hpo.jax.org/app/browse/term/HP:0031475 -- got it, I will make these changes!

**Comment by DLewisSmith** [comment]
My concern is that the old HPO terms had insufficiently specific definitions.

Sent from my phone

On 22 Feb 2020, at 16:50, Peter Robinson <notifications@github.com<mailto:notifications@github.com>> wrote:

﻿

@DLewisSmith<https://github.com/DLewisSmith>
@galerp<https://github.com/galerp>
@shiva-g<https://github.com/shiva-g>
@ihelbig<https://github.com/ihelbig>
@GBalagura<https://github.com/GBalagura>
@hughkearney<https://github.com/hughkearney>

Would it be appropriate to merge the new terms in this subhierarchy with the existing non-convulsive terms? This would mean changing the label and definition of the old term to e.g this current one. This would only be appropriate if we are talking about the same thing though, otherwise, we might have to put the old terms in a "Pre-ILAE2018 legacy seizure" hierarchy?

https://hpo.jax.org/app/browse/term/HP:0031475
